src/styles/global.css

```
@@ -1312,14 +1399,25 @@
   }
 
   .tag-row-mobile-scroll {
-    max-height: min(14rem, 28svh);
+    --filter-scroll-pill-height: 2.3rem;
+    --filter-scroll-row-gap: 0.75rem;
+    --filter-scroll-buffer: 8px;
+    max-height: calc(
+      (4 * var(--filter-scroll-pill-height)) +
```

Stylelint (17.10.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)